Atomic/backends/_ostree.py

```diff
@@ -150,7 +172,8 @@ def get_layers(self, image):
 
     @staticmethod
     def get_dangling_images():
-        return []
+        syscontainers = SystemContainers()
```

Comment:
Oh, the reason for that is cuz it is a static method, so I am assuming no self is needed?